_posts/en/newsletters/2019-08-07-newsletter.md

```diff
@@ -0,0 +1,148 @@
+---
+title: 'Bitcoin Optech Newsletter #58'
+permalink: /en/newsletters/2019/08/07/
+name: 2019-08-07-newsletter
+slug: 2019-08-07-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the maintenance release of Bitcoin Core
+0.18.1, summarizes a discussion about BIP174 extensions, and notes a
+proposal for LN trampoline payments.  Our *bech32 sending support*
+section this week features a special case study contributed by BRD and
+our *notable changes* section highlights several possibly significant
+developments.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Upgrade to Bitcoin Core 0.18.1:** this maintenance release makes
+  available several bug fixes and other improvements.  [Upgrading][bitcoin
```

Comment:
IIUC the link here points to https://bitcoincore.org/bin/bitcoin-core-0.18.1/. Perhaps a link to the release notes, e.g. behind "bug fixes and improvements" or "(release notes here)" etc., would be handy if not already present.